playground/src/main/res/drawable/ic_stopiteration.xml

```diff
@@ -0,0 +1,74 @@
+      android:fillColor="#000000"/>
```

Comment:
SVG's are too heavy, should be optimised via SVGO or redesigned, they will cause performance issues

Reply:
Done